exercises/hangman/README.md

```diff
@@ -0,0 +1,51 @@
+# Hangman
+
+Implement the logic of the hangman game using functional reactive programming.
+
+Hangman is a simple word guessing game.
+
+[Functional Reactive Programming][frp] is a way to write interactive
+programs. It differs from the usual perspective in that instead of
+saying "when the button is pressed increment the counter", you write
+"the value of the counter is the sum of the number of times the button
+is pressed."
+
+Implement the basic logic behind hangman using functional reactive
+programming.  You'll need to install an FRP library for this, this will
+be described in the language/track specific files of the exercise.
```

Comment:
This means we should add a HINT file. We can just say that _no_ library is necessary.

Reply:
I've reworded this sentence to indicate that no library is necessary, and added the word "optionally" with regard to FRP. Let me know what you think.